src/plot_hourly_bar_graph.py

```diff
@@ -3,38 +3,70 @@
 import datetime
 import csv
 import numpy
+import os
```

Comment:
For this file in general, pretty much the same comments as I made in the hourly_line_graph.py file

- use more descriptive variable names to make it more readable
- consider making a class for the raw data to create a list of raw data objects rather than weird stuff you have to index which makes it harder to read